### ISSUE TITLE
Feature Request: Quantized DepthwiseConv2dNative 

### ISSUE THREAD

**Issue by parvizp** [tensorflow/tensorflow#11670]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Linux Ubuntu 16.04
- **TensorFlow installed from (source or binary)**: Source
- **TensorFlow version (use command below)**: (master as of post) d3edb8c60ed4fd831d62833ed22f5c23486c561c
- **Python version**: 2.7
- **Bazel version (if compiling from source)**:
- **CUDA/cuDNN version**:
- **GPU model and memory**:
- **Exact command to reproduce**:

### Describe the problem
Models, such as MobileNet use _DepthwiseConv2dNative_. I couldn't find a quantized kernel and the quantization pass in _GraphTransform_ doesn't currently support DepthwiseConv2dNative: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/graph_transforms/quantize_nodes.cc#L57

Are there plans to provide a quantized version of DepthwiseConv2dNative?

**Comment by asimshankar** [comment]
I don't believe anyone is actively working on this at this time. 
@petewarden has more state on MobileNet and plans in this area, so he could confirm.

(FYI @suharshs )

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by suharshs** [comment]
We have focused our 8 bit fixed point efforts on TensorFlow Lite (https://www.tensorflow.org/mobile/tflite/). This means the quantized kernels are implemented as TFLite kernels. The current flow is to train "fake" quantized models (https://github.com/tensorflow/tensorflow/tree/master/tensorflow/contrib/quantize) and then convert them to TFLite models with TOCO, the TFLite converter (https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/lite/toco/g3doc/cmdline_examples.md).

If you are looking for quantized mobilenet_v1 models, we have pretrained checkpoints, corresponding TFLite buffers, frozen graphdefs, and training scripts available here: https://github.com/tensorflow/models/blob/master/research/slim/nets/mobilenet_v1.md

Hope that helps!
